src/util/channel_table.c

```diff
@@ -26,7 +26,7 @@ TableResponseStatus chan_table_init(int entry_num)
   
   charbuf *temp;
 
-  if ((temp = (charbuf *) realloc(chan_table.chan_key, entry_num * sizeof(charbuf))) == NULL)
+  if ((temp = (charbuf *) malloc(entry_num * sizeof(charbuf))) == NULL)
   {
     pelz_log(LOG_ERR, "Key List Space Reallocation Error");
```

Comment:
The error message and return code should change to reflect that this is no longer a reallocation.